app/views/about/terms.en.html.haml

```diff
@@ -2,75 +2,78 @@
-
-  %p We use cookies to understand and save your preferences for future visits and compile aggregate data about site traffic and site interaction so that we can offer better site experiences and tools in the future. We may contract with third-party service providers to assist us in better understanding our site visitors. These service providers are not permitted to use the information collected on our behalf except to help us conduct and improve our business.
-
-  %h3#disclose Do we disclose any information to outside parties?
-
-  %p We do not sell, trade, or otherwise transfer to outside parties your personally identifiable information. This does not include trusted third parties who assist us in operating our site, conducting our business, or servicing you, so long as those parties agree to keep this information confidential. We may also release your information when we believe release is appropriate to comply with the law, enforce our site policies, or protect ours or others rights, property, or safety. However, non-personally identifiable visitor information may be provided to other parties for marketing, advertising, or other uses.
-
-  %h3#third-party Third party links
-
-  %p Occasionally, at our discretion, we may include or offer third party products or services on our site. These third party sites have separate and independent privacy policies. We therefore have no responsibility or liability for the content and activities of these linked sites. Nonetheless, we seek to protect the integrity of our site and welcome any feedback about these sites.
-
-  %h3#coppa Children's Online Privacy Protection Act Compliance
-
-  %p
-    Our site, products and services are all directed to people who are at least 13 years old. If this server is in the USA, and you are under the age of 13, per the requirements of COPPA
-    = surround '(', '),' do
-      = link_to 'Children\'s Online Privacy Protection Act', 'https://en.wikipedia.org/wiki/Children%27s_Online_Privacy_Protection_Act'
-    do not use this site.
-
-  %h3#online Online Privacy Policy Only
-
-  %p This online privacy policy applies only to information collected through our site and not to information collected offline.
-
-  %h3#consent Your Consent
-
-  %p By using our site, you consent to our web site privacy policy.
-
-  %h3#changes Changes to our Privacy Policy
-
-  %p If we decide to change our privacy policy, we will post those changes on this page.
-
-  %p This document is CC-BY-SA. It was last updated May 31, 2013.
-
-  %p
-    Originally adapted from the
-    = succeed '.' do
-      = link_to 'Discourse privacy policy', 'https://github.com/discourse/discourse'
+  - if @instance_presenter.site_terms.blank?
```

Comment:
The issue here is that you are modifying `terms.en.html.haml` only (emphasis on "en"). Something must be shuffled around to make it `terms.html.haml` (locale-agnostic) and move the default text somewhere else. Either make the default value of the `site_terms` setting the English version of this in config/settings.yml, or put the whole text into a YML locale file (haml markup has to become html in both cases, of course)